service classifyServer{    
    string get_common_class(1:string text, 2:string class_dic)
    string get_emotion_class(1:string text)
}    
